test/log.spec.js

```diff
@@ -65,7 +56,7 @@ describe('.log', function () {
 
     expect(res).to.exist()
     expect(res).to.be.an('object')
-    expect(res).to.not.have.property('Error')
-    expect(res).to.have.property('Message')
+    expect(res).to.not.have.property('error')
+    expect(res).to.have.property('message')
```

Comment:
Did the property names change?

Reply:
Yep! ...and documented in the breaking changes above.